modules/dnn/perf/perf_layer.cpp

```diff
@@ -55,7 +55,123 @@ struct Layer_Slice : public TestBaseWithParam<tuple<Backend, Target> >
     }
 };
 
+struct Layer_NaryEltwise : public TestBaseWithParam<tuple<Backend, Target> >
```

Comment:
If we decide to leave  this performance test, we need to factor out common code and generate data randomly.

Reply:
Replaced the for loop value asignment with cv::randn which fills the mat with random numbers from standard normal distribution (mean=0 and std=1).